scenarios/mqtt-pravega-bridge/src/main/dist/conf/bridge.properties

```diff
@@ -0,0 +1,14 @@
+# Pravega Properties
+controllerUri=tcp://127.0.0.1:9090
+scope=examples
+stream=mqtt-example
```

Comment:
For a sample, writing to a single stream is ok, but in general, isn't it possible that we have use cases in which the same bridge writes to different streams, perhaps inferring the stream from the event data?